src/main/java/com/mathworks/ci/MatlabBuildStepExecution.java

```diff
@@ -25,20 +25,26 @@ public class MatlabBuildStepExecution extends SynchronousNonBlockingStepExecutio
     
     private String tasks;
     private String startupOptions;
+    private String buildOptions;
     private static String DEFAULT_PLUGIN = "+ciplugins/+jenkins/getDefaultPlugins.m";
     private static String BUILD_REPORT_PLUGIN = "+ciplugins/+jenkins/BuildReportPlugin.m";
     private static String TASK_RUN_PROGRESS_PLUGIN = "+ciplugins/+jenkins/TaskRunProgressPlugin.m";
 
-    public MatlabBuildStepExecution(StepContext context, String tasks, String startupOptions) {
+    public MatlabBuildStepExecution(StepContext context, String tasks, String startupOptions, String buildOptions) {
         super(context);
         this.tasks = tasks;
         this.startupOptions = startupOptions;
+        this.buildOptions = buildOptions;
     }
 
     private String getTasks() {
         return this.tasks;
     }
 
+    private String getBuildOptions() {
+        return this.buildOptions;
+    }
+
```

Comment:
Yes, valid point. I have removed the private getter for 'buildOptions' and for 'tasks' too, since we were just accessing the fields. Although, we may use the getters/setters in future if the implementation gets complex? - https://stackoverflow.com/questions/1568091/why-use-getters-and-setters-accessors